setup.cfg

```diff
@@ -4,7 +4,7 @@ exclude=migrations
 
 [coverage:run]
 branch = true
-omit = */tests*, */migrations/*, .tox/*, manage.py
+omit = */tests*, */migrations/*, .tox/*, manage.py, finance/credit.py
```

Comment:
I think we should keep the number of exclusions to a minimum and I don't think excluding this adds much value. It's not going to tip the % much either way. Knowing that it was never imported would also mean it was potentially dead code.